puppet.gemspec

```
@@ -1,24 +1,24 @@
 Gem::Specification.new do |spec|
-  spec.name = "puppet"
+  spec.name = "openvox"
   spec.version = "8.11.0"
   spec.licenses = ['Apache-2.0']
 
   spec.required_rubygems_version = Gem::Requirement.new("> 1.3.1")
   spec.required_ruby_version = Gem::Requirement.new(">= 3.1.0")
-  spec.authors = ["Puppet Labs"]
+  spec.authors = ["Vox Pupuli"]
   spec.date = "2012-08-17"
   spec.description = <<~EOF
-    Puppet, an automated administrative engine for your Linux, Unix, and Windows systems, performs administrative tasks
+    OpenVox is a community implementation of Puppet, an automated administrative engine for your Linux, Unix, and Windows systems, performs administrative tasks
     (such as adding users, installing packages, and updating server configurations) based on a centralized specification.
   EOF
-  spec.email = "info@puppetlabs.com"
+  spec.email = "pmc@voxpupuli.org."
```

We usually have the public ml here
```suggestion
  spec.email = "voxpupuli@groups.io"
```